src/microceph.py

```diff
@@ -83,6 +83,38 @@ def is_cluster_member(hostname: str) -> bool:
             raise e
 
 
+def bootstrap_cluster(params: dict):
+    """Bootstrap MicroCeph cluster."""
+    micro_ip = params.get("micro_ip", None)
+    public_net = params.get("public_net", None)
+    cluster_net = params.get("cluster_net", None)
```

Comment:
Minor style complaint: shorter and arguably more pythonic would be to have keyword args for this func. As a bonus you'd have type hints for the param values

```
def bootstrap_cluster(micro_ip: str = None, public_net: str = None, cluster_net: str = None):
...
```
And call it above with 

```
 microceph.bootstrap_cluster(**self._get_bootstrap_params())
```

Reply:
Neat!